packages/docs/src/data/api/rules/match-pattern.ts

```diff
@@ -0,0 +1,22 @@
+import { Api } from '~/types';
+import { ruleMessages } from '../shared/ruleMessages';
+
+export const api: Api = {
+	matchPattern: {
+		arguments: [
+			{
+				name: 'pattern',
+				type: 'RegExp',
+				description: 'Le motif à respecter.'
+			},
+			{
+				name: 'errorMessages',
+				type: 'ErrorMessages<number>',
+				description: 'Les messages d’erreur personnalisés.',
+			},
```

Comment:
```suggestion
			},
```
La fonction `ruleMessage` génère cette prop